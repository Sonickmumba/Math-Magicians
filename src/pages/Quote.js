import React from 'react';

const Quote = () => (
  <p className="math-quote">
    Mathematics is not about numbers, equations, computations, or algorithms:
    Its about understanding. - Williams Paul Thurston
  </p>
);

export default Quote;
