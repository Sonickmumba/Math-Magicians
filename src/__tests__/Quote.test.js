import { render, screen } from '@testing-library/react';
import TestRenderer from 'react-test-renderer';
import Quote from '../pages/Quote';

test('Test for Quote Component', () => {
  render(<Quote />);
  const linkElement = screen.getByText(/Mathematics is not about numbers/i);
  expect(linkElement).toBeInTheDocument();
});

describe('Test for Quote component', () => {
  it('Quote Component renders correctly', () => {
    const tree = TestRenderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
