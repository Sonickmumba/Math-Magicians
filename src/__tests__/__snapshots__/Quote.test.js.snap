// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for Quote component Quote Component renders correctly 1`] = `
<p
  className="math-quote"
>
  Mathematics is not about numbers, equations, computations, or algorithms: Its about understanding. - Williams Paul Thurston
</p>
`;
